packages/owlbot-bootstrapper/cli/run-trigger-command.ts

```diff
@@ -27,6 +27,7 @@ export interface CliArgs {
   monoRepoDir?: string;
   serviceConfigPath?: string;
   interContainerVarsPath?: string;
+  skipIssueOnFailure: string;
```

Comment:
shouldn't this be a boolean?

Reply:
It needs to be converted into a string to be an env variable. I added an option to be a boolean as well.